docs/modules/api_gateway.md

```diff
@@ -21,16 +21,17 @@ module "api_gateway" {
   tags                  = {}
   log_retention_in_days = 0
 
-  # Specify lambdas by (arbitrary) key and function-name for later reference via a route.
-  # The function ARN and invocation ARNs also have to be specified.
-  lambdas = {
+  authorizers = {
     "authorizer_lambda" = {
-      "function_name"       = module.lambda_authorizer.lambda_function_name
       "function_arn"        = module.lambda_authorizer.lambda_function_arn
       "function_invoke_arn" = module.lambda_authorizer.lambda_function_invoke_arn
     }
+  }
+
+  # Specify lambdas by (arbitrary) key and function-name for later reference via a route.
+  # The function ARN and invocation ARNs also have to be specified.
```

Comment:
This comment's out of date now that `function_name` isn't required

Reply:
Fixed